R/DataSetList.R

```diff
@@ -159,6 +163,15 @@ DataSetList <-
       attr(object, 'funcId') <- funcId
       attr(object, 'algId') <- algId
     }
+    
+    suite <- unique(suites)
```

Comment:
Should we make sure that there is only one suite in ONE DataSetList? And we should prevent combining two DataSetLists that have different `suite`?

Reply:
Yes, that is being done right now, but now it just gives a warning and not an error. I will change it to an error instead